pkgs/by-name/fr/framework-tool/package.nix

```diff
@@ -5,13 +5,13 @@ rustPlatform.buildRustPackage {
 
   # Latest stable version 0.1.0 has an ssh:// git URL in Cargo.lock,
   # so use unstable for now
-  version = "unstable-2023-11-14";
+  version = "unstable-2024-06-14";
```

Comment:
Unstable versions should include the last version the source commits are based on, check https://github.com/NixOS/nixpkgs/blob/31fcbf0c46c282ced203d0faff663e3fb87efac1/pkgs/README.md#package-naming .